byway/src/main/java/com/google/sps/servlets/CreateTripServlet.java

```diff
@@ -0,0 +1,50 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceConfig;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.common.flogger.FluentLogger;
+import com.google.gson.Gson;
+import com.google.sps.data.Trip;
+import com.google.sps.data.UserInfo;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/api/createtrip")
+public class CreateTripServlet extends HttpServlet {
+  private static final FluentLogger logger = FluentLogger.forEnclosingClass();
```

Comment:
Can remove this if you don't need to log anything.

Reply:
done